mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -35,8 +36,29 @@
 
 
 @testing.requires_testing_data
+@requires_h5py
 def test_io_set():
     """Test importing EEGLAB .set files."""
+    _test_io_set(False)
+    _test_io_set(True)
+
+
+@testing.requires_testing_data
+@requires_h5py
+def _test_io_set(use_hdf=True):
+    """Test importing EEGLAB .set files."""
+
+    if use_hdf:
+        raw_fname = op.join(base_dir, 'test_raw_h5.set')
+        raw_fname_onefile = op.join(base_dir, 'test_raw_onefile_h5.set')
+        epochs_fname = op.join(base_dir, 'test_epochs_h5.set')
+        epochs_fname_onefile = op.join(base_dir, 'test_epochs_onefile_h5.set')
+    else:
+        raw_fname = op.join(base_dir, 'test_raw.set')
+        raw_fname_onefile = op.join(base_dir, 'test_raw_onefile.set')
+        epochs_fname = op.join(base_dir, 'test_epochs.set')
+        epochs_fname_onefile = op.join(base_dir, 'test_epochs_onefile.set')
```

Comment:
What I suggest is to put all of filenames (old format and new format) above, around line 34. It's probably worth renaming the old variables to e.g. `raw_fname_old` or `raw_fname_legacy` for the old/legacy MATLAB format. Then here you do:
```
if use_hdf:
    raw_fname = raw_fname_h5
    ...
else:
    raw_fname = raw_fname_old
    ...
```